### PR TITLE
update the Helm ref

### PR DESCRIPTION
**Pull request by rosemarymarano**
Update the Helm reference doc as per [OWLS-95907](https://jira.oraclecorp.com/jira/browse/OWLS-95907): Operator User Guide Feedback and Updates for 4.0. also, minor edits elsewhere to conform to Oracle Style Guide.

### CODE REVIEW

File: documentation/staging/content/userguide/managing-operators/using-helm.md [hunk 1]
```diff
@@ -393,39 +389,24 @@ This value is required if `domainNamespaceSelectionStrategy` is `RegExp` and ign
...
 - If your operator Helm `enableClusterRoleBinding` configuration value is `false`, then
   then a running operator will _not_ have privilege to manage a newly added namespace
   that matches its regular expression until you upgrade
   the operator's Helm release.
   See [Ensuring the operator has permission to manage a namespace]({{< relref "/userguide/managing-operators/namespace-management#ensuring-the-operator-has-permission-to-manage-a-namespace" >}}).
-- If the deprecated `dedicated` setting is set to `true`,
-  then this value is ignored and
-  the operator will manage only domains in its own namespace.
-
-##### `dedicated` ***(Deprecated)***
```
Comment (tbarnes-us) [inline documentation/staging/content/userguide/managing-operators/using-helm.md:407]: IMO, it is only appropriate to remove `dedicated` if Operator 4.0 is not going to support it. Is that the case?

Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-operators/using-helm.md:407]: Yes.

**Review by tbarnes-us — APPROVED** [review]
Approved, but did leave one comment with a question.

File: documentation/staging/content/userguide/managing-operators/using-helm.md (hunk 1 above)
Comment (robertpatrick) [reply documentation/staging/content/userguide/managing-operators/using-helm.md:407]: @tbarnes-us Rosemary is doing what I asked her to do.  This parameter has been deprecated for a long time now, we should remove it completely in 4.0.

**Review by rjeberhard — APPROVED** [review]

**Review by robertpatrick — APPROVED** [review]
One minor typo

File: documentation/staging/content/userguide/managing-operators/installation.md [hunk 2]
```diff
@@ -187,7 +188,7 @@ When upgrading the operator:
 - Use the `helm upgrade` command with the `--reuse-values` parameter.
 - Supply a new `image` value.
 
-This is because, even with a new version of the Helm chart,
+This rationale for this is because, even with a new version of the Helm chart,
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-operators/installation.md:191]: @robertpatrick OH. Thanks. I'll fix it in my next PR.

File: documentation/staging/content/userguide/managing-operators/using-helm.md (hunk 1 above)
Comment (tbarnes-us) [reply documentation/staging/content/userguide/managing-operators/using-helm.md:407]: Rosemary filled me in above. Thanks.

**Pull request merged**